tox.ini

```diff
@@ -441,13 +416,11 @@ commands =
 whitelist_externals = /bin/sh
 skipsdist = True
 deps =
-    pylint==2.11.1
+    tomte[pylint]==0.1.1
     pytest==7.0.0
```

Comment:
TODO: Let's create a dedicated tox env for package deps so we can reduce this to a single line here